src/SideBar/useReportsFeed/index.js

```diff
@@ -19,8 +19,8 @@ const useReportsFeed = () => {
   const events = useSelector((state) => getFeedEvents(state));
   const userIsGeoPermRestricted = useSelector((state) => userIsGeoPermissionRestricted(state?.data?.user));
   const userLocationCoords = useSelector((state) => state?.view?.userLocation?.coords);
+  const feedSort = eventFilter.filter.sort ?? DEFAULT_EVENT_SORT;
```

Comment:
I know this line should be replaced by setting the default value directly into `INITIAL_FILTER_STATE` but it was causing a bunch of errors due the `DEFAULT_EVENT_SORT` not being initialized when using the initial state, in order to solve that I should have to move around several declarations and impacting a lot of files where those consts were being imported I just wanted to avoid all those changes but not sure, If you want me I can go ahead and do that 🫡

Reply:
It seems like you could just move some of the `const`s from the event filter utils file to somewhere initialized sooner and it wouldn't be that much hassle to me? Unless I'm missing something here.

Reply:
Don't we have a constants file?